doc/source/ci-cd/github-actions.rst

```diff
@@ -0,0 +1,50 @@
+==============
+GitHub-Actions
+==============
+
+Here is a minimal complete example workflow to create a Code Scanning action using Bandit by using GitHub Actions from PyCQA's `bandit-action <https://github.com/PyCQA/bandit-action>`.
+
+==============
+Example YAML code for GitHub Actions Pipeline
+==============
```

Comment:
Instead of `=` should be `-` which are the same length as `Example YAML code for GitHub Actions Pipeline`